### ISSUE TITLE
Velocity loss

### ISSUE THREAD

**Issue by jotaf98** [FangGet/PackNet-SFM-PyTorch#5]
Hi, thanks for sharing your implementation!

I was just wondering, why is the velocity loss (in trainer.py) commented out? Is it that it doesn't seem to work as well as the paper claims yet?

**Comment by FangGet** [comment]
No，I add the part for velocity loss and it seems work well for now, it's commented out as I'd like to use only unsupervised part when it's uploaded.

You can definitely try uncomment it and train with it.

**Comment by jotaf98** [comment]
OK, that's good to know!

I guess at some point it could be turned into a command-line option, but given your answer it's not a big deal.
